packages/front-end/components/Experiment/TabbedPage/HealthTab.tsx

```diff
@@ -75,10 +75,21 @@ export default function HealthTab({
             <Button
               className="mt-2 mb-2 ml-auto"
               style={{ width: "200px" }}
-              onClick={async () => await enableRunHealthTrafficQueries()}
+              onClick={async () => setUiMode("setup")}
             >
               Enable Health Queries
             </Button>
+            {uiMode === "setup" ? (
+              <HealthTabOnboardingModal
+                experiment={experiment}
+                phase={phase}
+                close={() => setUiMode("open")}
+                refreshOrganization={refreshOrganization}
+                mutateSnapshot={mutateSnapshot}
+                setAnalysisSettings={setAnalysisSettings}
+                setLoading={setLoading}
+              />
+            ) : null}
           </>
```

Comment:
Expose `open` from the `Modal` component this is wrapping and pass in the state you're using (`uiMode`) so the `Modal` can handle hiding on its own

Reply:
done ty